apps/docs/content/guides/queues/queue-management.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: 'SQL management'
+description: 'Manage your Queues using SQL'
+---
+
+You can manage your Queues using the Dashboard UI, or via SQL. See the [Quickstart](/guides/queues/quickstart) for Dashboard instructions.
+
+The following sections describe the SQL API for managing Queues, which uses [pgmq](https://github.com/tembo-io/pgmq) under the hood. Run the SQL in the [SQL Editor](/dashboard/project/_/sql/new) or via a Postgres client.
+
+## Install the Queues Module
+
+```sql
+create schema if not exists pgmq;
+create extension if not exists pgmq with schema pgmq;
```

Comment:
And if they do, do we want to give them instructions to wrap the pgmq functions into the pgmq_public schema?

This will change the instructions for how users call the functions in sql